app/views/layout.njk

```diff
@@ -53,8 +53,8 @@
 {% endblock %}
 
 {% block content %}
-  {% if errorList %}
-    {{ govukErrorSummary({ titleText: "There is a problem", errorList: errorList }) }}
+  {% if error %}
+    {{ govukErrorSummary({ titleText: "There is a problem", errorList: error.errorList }) }}
```

Comment:
should this be error.summary instead of error.errorList as it is specific to this component ? 

Or errorSummaryList

Reply:
If pushed, I'd go with `errorSummaryList`. Or even `error.summary.errorList` (adds scope to allow us to pass other properties to the summary in the future).

However, I prefer to keep it as `errorList` for now because it matches the name of the property we're populating. 🤷